resources/views/admin/index.blade.php

```diff
@@ -52,6 +54,16 @@
 
 @push('scripts')
     <script defer>
-        jQuery('table').DataTable();
+        jQuery('table').DataTable({
+            stateSave: true,
+            columnDefs: [
+                {
+                    // Hide the notes field, but leave it searchable.
+                    targets: [ 8 ],
```

Comment:
Presume this means you can only target by index, and we'd need to remember this in case we move fields around?

Reply:
Yes, that's the way it looks.